<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportMetadataDialog</class>
 <widget class="QDialog" name="ImportMetadataDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import metadata</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="editMetadataFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Choose a metadata file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btBrowseMetadataFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/folder-open.svg</normaloff>:/icons/images/folder-open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="6" colspan="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QCheckBox" name="chkUseFirstLineAsHeader">
        <property name="text">
         <string>Use first line as column header</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkComment">
        <property name="text">
         <string>Ignore lines starting with</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="editComment">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CsvDelimiterCombo" name="cbCsvDelimiter"/>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinSkipRows">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="editCsvDelimiter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Separator</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start import row</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Index column</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cbIndexColumn"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btSelectAll">
          <property name="toolTip">
           <string>Select all items</string>
          </property>
          <property name="text">
           <string>Select &amp;all</string>
          </property>
          <property name="icon">
           <iconset resource="ui.qrc">
            <normaloff>:/icons/images/select-all.svg</normaloff>:/icons/images/select-all.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btSelectNone">
          <property name="toolTip">
           <string>Select nothing</string>
          </property>
          <property name="text">
           <string>Select &amp;none</string>
          </property>
          <property name="icon">
           <iconset resource="ui.qrc">
            <normaloff>:/icons/images/select-none.svg</normaloff>:/icons/images/select-none.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btSelectInvert">
          <property name="toolTip">
           <string>Invert selection</string>
          </property>
          <property name="text">
           <string>&amp;Invert selection</string>
          </property>
          <property name="icon">
           <iconset resource="ui.qrc">
            <normaloff>:/icons/images/select-invert.svg</normaloff>:/icons/images/select-invert.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btRefresh">
          <property name="toolTip">
           <string>Refresh preview</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ui.qrc">
            <normaloff>:/icons/images/refresh.svg</normaloff>:/icons/images/refresh.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="LoadingTableWidget" name="twMetadata"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CsvDelimiterCombo</class>
   <extends>QComboBox</extends>
   <header>lib/ui/widgets.h</header>
  </customwidget>
  <customwidget>
   <class>LoadingTableWidget</class>
   <extends>QTableWidget</extends>
   <header>lib/ui/widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportMetadataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportMetadataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
